tensorboard/plugins/graph/tf_graph_dashboard/tf-graph-dashboard.html

```diff
@@ -145,9 +142,33 @@ <h3>No graph definition files were found.</h3>
  */
 const RUN_STORAGE_KEY = 'run';
 
+/**
+ * TODO(stephanwlee): Convert this to proper type when converting to TypeScript.
+ * @typedef {{
+ *   tag: ?string,
+ *   displayName: string,
+ *   conceptualGraph: boolean,
+ *   opGraph: boolean,
+ *   profile: boolean,
+ * }}
+ */
+ const TagItem = {};
+
+/**
+ * TODO(stephanwlee): Convert this to proper type when converting to TypeScript.
+ * @typedef {{
+ *   name: string,
+ *   tags: Arary<!TagItem>,
+ * }}
+ */
+const RunItem = {};
+
 Polymer({
   is: 'tf-graph-dashboard',
   properties: {
+    /**
+     * Array<!RunItem>
```

Comment:
AFAIK you have to use `@type` for the compiler to actually pick up this
annotation.

Reply:
Done. I am not actually sure our Closure compiler understands these Polymer props. I have not seen it working, yet.